### DISCUSSION TITLE
Installing issues on MacBook with M2 on both Parallel and virtualbox

### DISCUSSION THREAD

**Discussion by Akin-Omoge** [Security-Onion-Solutions/securityonion#13845] [2.4]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

2

### RAM

4

### Storage for /

200

### Storage for /nsm

32

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

here is the screenshot of the landing page after i configure the VM

### Guidelines

- [ ] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Akin-Omoge** [comment]
<img width="1231" alt="Screenshot 2024-10-20 at 9 29 38 AM" src="https://github.com/user-attachments/assets/19222423-cebb-47c2-b1ba-01266269647d">

**Comment by Akin-Omoge** [comment]
Here is the error massage/landing page 
<img width="1231" alt="Screenshot 2024-10-20 at 9 29 38 AM" src="https://github.com/user-attachments/assets/a5e7894e-5b9a-41a9-800b-8b55bb3b1d74">

**Comment by Akin-Omoge** [comment]
<img width="861" alt="Screenshot 2024-10-20 at 9 42 46 AM" src="https://github.com/user-attachments/assets/d2feb135-cd60-47a8-aa45-6688e126366a">

**Comment by dougburks** [comment]
Please see the documentation:
https://docs.securityonion.net/en/2.4/hardware.html#cpu-architecture
